src/utils/decorators/input-label/__spec__.js

```diff
@@ -295,4 +295,35 @@ describe('InputLabel', () => {
       expect(instanceFalse.mainClasses).toEqual('');
     });
   });
+
+  describe('displayName', () => {
+    class Foo extends React.Component { // eslint-disable-line react/no-multi-comp
+      bar = () => {
+        return 'bar';
+      }
+    }
+
+    const displayName = 'FooClass';
+
+    describe('when ComposedComponent.displayName is defined', () => {
```

Comment:
[minor] We could probably move these specs into something more generic and pass in the decorator. e.g. how the data attribute specs are done